src/processing/script/transformers/eval-get.js

```diff
@@ -16,32 +17,40 @@ export default {
 
     nodeTypes: [Syntax.Identifier, Syntax.MemberExpression],
 
-    condition: (node, parentNode) => {
+    condition: (node, parent) => {
```

Comment:
I think it's will be much cleaner if we'll have two separate transforms: one for the identifier and one for the member. You can call them `eval-get` and `window-eval-get` respectively. @churkin What do you think?